src/styles/navigation.css

```diff
@@ -0,0 +1,236 @@
+      <div class="container horizontal middle">
+        <h2 class="title heading-2">Heading</h2>
+        <ul class="nostyle contents">
+          <li class="nav-item active">Item 1</li>
+          <li class="nav-item">Item 2</li>
+          <li class="nav-item active">Item 3</li>
+        </ul>
+      </div>
+    </nav>
+    <nav class="third-level">
+      <div class="container horizontal middle">
+        <h2 class="title heading-2">Heading</h2>
+        <ul class="nostyle contents">
+          <li class="nav-item">Item 1</li>
+          <li class="nav-item active">Item 2</li>
+          <li class="nav-item">Item 3</li>
+        </ul>
+      </div>
+    </nav>
+*/
+
+
+nav.top-level, nav.second-level, nav.third-level {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+}
+
+nav.top-level .nav-item, nav.second-level .nav-item, nav.third-level .nav-item {
+  cursor: pointer;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+nav.top-level ul, nav.second-level ul, nav.third-level ul {
+  display: flex;
+  align-items: center;
+}
+
+nav.top-level > .container, nav.second-level > .container, nav.third-level > .container {
+  grid-column-start: 2;
+  grid-column-end: 12;
+}
+
+nav.second-level > .container > .title, nav.third-level > .container > .title {
+  width: 140px;
+  word-break: break-word;
+}
+
+nav.top-level {
+  height: 40px;
+  color: var(--color-white);
+  background-color: var(--color-primary);
+}
+
+nav.top-level .title {
+  padding-right: 30px;
+}
+ 
+nav.top-level .nav-item {
+  padding: 10px 20px;
+  color: var(--color-white);
+}
+
+nav.top-level .nav-item.active, 
+nav.top-level .nav-item:hover,
+nav.top-level .nav-item:focus {
+  background-color: var(--color-primary-g1);
+}
+
+nav.top-level .nav-item {
+  padding: 0 20px;
+  color: var(--color-white);
+  font-weight: 200px;
+  font-size: 14px;
+}
+
+nav.second-level {
+  background-color: var(--color-accent-g2);
+  height: 80px;
+}
+
+nav.second-level:after {
+  content: ' ';
+  height: 0;
+  width: 100%;
+  display: block;
+  position: relative;
+  box-shadow: 0 0 3px 1px var(--color-alto);
+}
+
+nav.second-level .nav-item, nav.third-level .nav-item {
+  padding: 0 25px;
+  text-transform: uppercase;
+}
+
+nav.second-level .nav-item {
+  position: relative;
+}
+
+nav.second-level .nav-item:after {
+  content: ' ';
+  width: 0;
+  height: 0;
+  border-left: 10px solid transparent;
+  border-right: 10px solid transparent;
+  border-bottom: 10px solid transparent;
+  position: absolute;
+  left: calc(50% - 16px);
+  bottom: 0;
+  z-index: 3;
+  transition: border-bottom-color 200ms ease-in;
+}  
+
+nav.second-level .nav-item.active, 
+nav.second-level .nav-item:focus, 
+nav.second-level .nav-item:hover {
+  background-color: transparent;
+  color: var(--color-primary);
+}
+
+nav.second-level .nav-item.active .badge, 
+nav.second-level .nav-item:focus .badge, 
+nav.second-level .nav-item:hover .badge {
```

Comment:
are there examples of markup for this we could include?

Reply:
scroll to the top of the file :)

Reply:
i did see the example markup at the top of the file, but i didn't see it giving examples of using badges specifically and was curious about that.

definitely agreed that detailed usage of icons shouldn't be handled here, i was more asking about whether the current structure can handle icons provided from somewhere else in our application code (not particularly important for this PR, was just a curious question).

Reply:
👍 I see. Yeah, I did not include examples with icons. I will add more when I integrate the bootstrap hamburger responsiveness to it.

Reply:
I removed all the badges for now. I will start integrating this in one of our pages and I think I will get a better idea of what works. Thank you!